force-app/main/default/lwc/cloneDocumentCmp/cloneDocumentCmp.html

```diff
@@ -6,6 +6,9 @@
         <div hidden>
             <lightning-input-field field-name="DxCPQ__Parent_Template__c"> </lightning-input-field>
         </div>
+        <div hidden>
```

Comment:
Is it necessary for the div to be hidden rather than removing it completely unless its being referenced in the JS. Since no data-id / key is being used.